### PR TITLE
(maint) Make CA CLI release note point to the right docs

### PR DESCRIPTION
**Pull request by Magisus**

### CODE REVIEW

File: documentation/release_notes.markdown [hunk 1]
```diff
@@ -19,7 +19,7 @@ This Puppet Server release provides a new workflow and API for certificate issua
 
 ### New features
 
-- There is now a CLI command for setting up the certificate authority, called `puppetserver ca`. See [Puppet Server: Intermediate CA](/puppetserver/latest/intermediate_ca.html) for more information. [(SERVER-2172](https://tickets.puppetlabs.com/browse/SERVER-2172))
+- There is now a CLI command for setting up the certificate authority, called `puppetserver ca`. See [Puppet Server: Subcommands](/puppetserver/latest/subcommands.html) for more information. [(SERVER-2172](https://tickets.puppetlabs.com/browse/SERVER-2172))
 
 - The Puppet master's cert is now authorized to connect to the `certificate-status` endpoint out of the box. This allows the new CA CLI tool to perform CA tasks via Puppet Server's CA API. ([SERVER-2308](https://tickets.puppetlabs.com/browse/SERVER-2308))
```
Comment (justinstoller) [inline documentation/release_notes.markdown]: This wording is a little off, can we fix it up as well?

File: documentation/release_notes.markdown [hunk 2]
```diff
@@ -19,7 +19,7 @@ This Puppet Server release provides a new workflow and API for certificate issua
 
 ### New features
 
-- There is now a CLI command for setting up the certificate authority, called `puppetserver ca`. See [Puppet Server: Intermediate CA](/puppetserver/latest/intermediate_ca.html) for more information. [(SERVER-2172](https://tickets.puppetlabs.com/browse/SERVER-2172))
```
Comment (justinstoller) [inline documentation/release_notes.markdown]: We still might want to link to the intermediate ca docs somewhere? If not a mention about this here, then maybe below in the bug fix section were we talk about intermediate CA support?

File: documentation/release_notes.markdown (hunk 1 above)
Comment (justinstoller) [reply documentation/release_notes.markdown]: Like that this is only on fresh installs and that upgrades need to follow [link to upgrade docs about whitelisting masters old cert].

**Comment by Magisus** [comment]
Updated.

**Pull request merged**